packages/client/hmi-client/src/workflow/ops/stratify-mira/tera-stratify-mira.vue

```diff
@@ -276,29 +290,17 @@ const inputChangeHandler = async () => {
 	}
 };
 
-const saveNewModel = async (modelName: string, options: SaveOptions) => {
-	if (!amr.value || !modelName) return;
-	amr.value.header.name = modelName;
+const saveNewModel = async (modelName: string) => {
+	if (!stratifiedAmr.value || !modelName) return;
+	stratifiedAmr.value.header.name = modelName;
 
 	const projectResource = useProjects();
-	const modelData = await createModel(amr.value);
+	const modelData = await createModel(stratifiedAmr.value);
 	const projectId = projectResource.activeProject.value?.id;
 
 	if (!modelData) return;
-
-	if (options.addToProject) {
-		await projectResource.addAsset(AssetType.Model, modelData.id, projectId);
-	}
-
-	if (options.appendOutputPort) {
-		emit('append-output', {
-			id: uuidv4(),
-			label: modelName,
-			type: 'modelId',
-			value: [modelData.id]
-		});
-		emit('close');
-	}
+	await projectResource.addAsset(AssetType.Model, modelData.id, projectId);
+	useToastService().success('', `Saved to project ${projectResource.activeProject.value?.name}`);
```

Comment:
Please use `logger.info()` or add a new `LogLevels` for success.

Reply:
Why? There is no need to log this event.

Reply:
Once [my PR](https://github.com/DARPA-ASKEM/terarium/pull/2759) is approved you can do the following:
`logger.success(`Saved to project ${projectResource.activeProject.value?.name}`, { silent: true });`
This will display it green and do not send the log to the back-end.

We want to log everything to the back-end to be able to do a "detailled report" if necessary in the future.

Reply:
Cannot use `logger.success` because it maps to `console[level]` and `console["success"]` does not exist.